collect/get-logging-paths.js

```diff
@@ -5,7 +5,11 @@ const path = require('path')
 function getLoggingPaths (options) {
   let dirpath, basename
   if (options.hasOwnProperty('identifier')) {
-    dirpath = ''
+    if (options.hasOwnProperty('path') && options.path) {
+      dirpath = options.path
+    } else {
+      dirpath = './'
```

Comment:
this should not be needed and I would prefer it to stay as `''`.

Reply:
I think having an explicit path makes the code more readable. Why would you use `''` instead of `'./'`?

Reply:
I am afraid of Windows.

Reply:
Anyway, CI is passing, so +1.